content/en/_index.html

```diff
@@ -1,16 +1,67 @@
 +++
 title = "Tekton"
-linkTitle = "Tekton"
+linkTitle = "Tekton.dev Homepage"
 
 +++
 
-{{< blocks/cover title="Kubernetes-native CI/CD" image_anchor="top" height="med" color="orange" >}}
+{{< blocks/cover title="Kubernetes-native CI/CD" subtitle="" image_anchor="smart" color="orange" height="med">}}
 <div class="mx-auto">
+	<a class="btn btn-lg btn-dark mr-3 mb-4" href="{{< relref "/docs" >}}">
+		Documentation <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+	</a>
+	<a class="btn btn-lg btn-light mr-3 mb-4" href="https://github.com/tektoncd">
+		GitHub <i class="fab fa-github ml-2 "></i>
+	</a>
 </div>
 {{< /blocks/cover >}}
 
+{{% blocks/showcase color="link" %}}
 
-{{% blocks/lead color="primary" %}}
-This site is currently under construction. [You can find more information
-about Tekton here](https://github.com/tektoncd/).
-{{% /blocks/lead %}}
+{{< blocks/section color="danger" >}}
+{{% blocks/feature icon="fas fa-globe-asia" title="Standardization" %}}
+Tekton standardizes CI/CD tooling and processes accorss vendors, languages,
+and deployment environments. It works well with Jenkins, Jenkins X, Skaffold,
+Knative, and many other popular CI/CD tools.
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fas fa-medal" title="Built-in best practices" %}}
+Tekton let you create cloud-native CI/CD systems quickly, leveraging advanced
```

Comment:
Minor typo `Tekton let you` should be `Tekton lets you`... OR consider re-using the wording from the similar sections on the documentation page.

Reply:
Done. Thanks!